src/main/java/com/faforever/client/chat/OfficialLanguageChannel.java

```diff
@@ -0,0 +1,35 @@
+package com.faforever.client.chat;
+
+import java.util.Optional;
+
+public enum OfficialLanguageChannel {
+  FRENCH("fr", "#french"),
+  GERMAN("de", "#german"),
+  RUSSIAN("ru", "#russian");
+
+  private final String localeLanguage;
+  private final String channelName;
+
+
+  OfficialLanguageChannel(String localeLanguage, String channelName) {
+    this.localeLanguage = localeLanguage;
+    this.channelName = channelName;
+  }
+
+  public static Optional<OfficialLanguageChannel> getChannelName(String localeLanguage) {
+    for (OfficialLanguageChannel channel : values()) {
+      if (channel.getLocaleLanguage().equals(localeLanguage)) {
```

Comment:
Python client also adds players with 'by' locale to #russian, 'uk'  is a candidate too.

Reply:
@axel1200 Should i change the code so that it can deal with multiple locales per language channel? I don't think that would be a big change.